code/modules/power/power.dm

```diff
@@ -118,6 +118,40 @@
 	local_apc.cell.use(amount)
 	return TRUE
 
+/**
+  * Attemps to draw power directly from the APC's powernet rather than the APC's battery. For high-draw machines, like the cell charger
+  *
+  * Checks the surplus power on the APC's powernet, and compares to the requested amount. If the requested amount is available, this proc
+  * will add the amount to the APC's usage and return that amount. Otherwise, this proc will return FALSE.
+  * If the take_any var arg is set to true, this proc will use and return any surplus that is under the requested amount, assuming that
+  * the surplus is above zero.
+  * Args:
+  * - amount, the amount of power requested from the powernet. In standard loosely-defined SS13 power units.
+  * - take_any, a bool of whether any amount of power is acceptable, instead of all or nothing. Defaults to FALSE
```

Comment:
```suggestion
  * Attempts to draw power directly from the APC's Powernet rather than the APC's battery. For high-draw machines, like the cell charger
  *
  * Checks the surplus power on the APC's powernet, and compares to the requested amount. If the requested amount is available, this proc
  * will add the amount to the APC's usage and return that amount. Otherwise, this proc will return FALSE.
  * If the take_any var arg is set to true, this proc will use and return any surplus that is under the requested amount, assuming that
  * the surplus is above zero.
  * Args:
  * - amount, the amount of power requested from the Powernet. In standard loosely-defined SS13 power units.
  * - take_any, a bool of whether any amount of power is acceptable, instead of all or nothing. Defaults to FALSE
```